docs/journalist.rst

```diff
@@ -564,6 +567,12 @@ are important steps you can take to protect yourself:
    (potentially including GPS coordinates) to anyone who gains access
    to the file.
 
+.. warning::
+
+   For maximum safety, it is recommended that cell phones and personal 
+   electronic devices be left outside of the secure environment where the
+   *Secure Viewing Station* is used.
```

Comment:
I think it's a good recommendation, but I will note that to decrypt a file, you'll have to unlock both the Transfer Device and the SVS -- which ideally will have separate passphrases. If folks don't have those passphrases memorized, they'll need to access them from somewhere, and that's most likely another electronic device.

Leaving this open for a bit in case other folks have thoughts on this specific recommendation and how to best word it.

Reply:
@nathandyer I would suggest rewording this as follows:
> If you have not memorized the passphrases to unlock the USB drives for the *Secure Viewing Station* or the *Transfer Device*, you may need to access a password manager on your phone or laptop to do so. We recommend switching any required electronic devices into airplane mode, and securely storing any devices you do *not* need outside the environment in which you access the *Secure Viewing Station*. This further mitigates the risk of accidentally compromising the air-gap.

Reply:
Thanks @eloquence! I like the suggested rewording, and have gone ahead and committed that change.